### PR TITLE
Do not reference Docker Hub anymore

### PR DESCRIPTION
**Pull request by ryanwohara**
# Description
Replacing Docker Hub with Azure, but not being overly specific because public images are still hosted in Docker Hub.

# Reasons
https://circleci.atlassian.net/browse/SERVER-2007

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: https://circleci.com/docs/style-guide-overview/.

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/server/installation/phase-2-core-services.adoc [hunk 1]
```diff
@@ -33,12 +33,12 @@ kubectl create ns <namespace>
 
 TIP: Once you have created your namespace, we recommend setting your `kubectl` context too, with the following command: `kubectl config set-context --current --namespace <namespace>`
 
-[#pull-image-from-dockerhub]
-== 2. Pull image from DockerHub
+[#pull-image-from-azure]
+== 2. Pull Images
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/installation/phase-2-core-services.adoc:37]:
```suggestion
[#pull-images]
== 2. Pull images
```

File: jekyll/_cci2/server/installation/phase-2-core-services.adoc [hunk 2]
```diff
@@ -33,12 +33,12 @@ kubectl create ns <namespace>
...
 
 Credentials to pull the images from CircleCI's image registry will be provided to you as part of the onboarding process. A `docker-registry` Kubernetes secret will be used to pull images from Azure Container Registry (ACR). You have two options, depending on whether your application has access to the public internet.
 
-[.tab.dockerhub.Public]
+[.tab.azure.Public]
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/installation/phase-2-core-services.adoc:41]:
```suggestion
[.tab.pullimage.Public]
```

File: jekyll/_cci2/server/installation/phase-2-core-services.adoc [hunk 3]
```diff
@@ -54,7 +54,7 @@ kubectl create secret docker-registry regcred \
 ----
 --
 
-[.tab.dockerhub.Private]
+[.tab.azure.Private]
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/installation/phase-2-core-services.adoc:57]:
```suggestion
[.tab.pullimage.Private]
```

**Review by rosieyohannan — COMMENTED** [review]
just removing all reference to where they are stored

**Comment by ryanwohara** [comment]
Brilliant, thank you

**Review by atulsingh0 — APPROVED** [review]

**Review by lloydChris — APPROVED** [review]

**Pull request merged**